meshio/xdmf/main.py

```diff
@@ -333,6 +337,10 @@ def __init__(
 
         xdmf_file = ET.Element("Xdmf", Version="3.0")
 
+        if info:
```

Comment:
This needs a read method, too.

Reply:
It is already there in `read_xdmf2()` and `read_xdmf3()`.